src/index.html

```diff
@@ -8,7 +8,205 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <div id="header" class="header-wrapper">
+        <ul id="main-nav">
+          <li class="main-nav-item"><a href="">Home</a></li>
+        </ul>
+        <div id="nav-side-btn">
+          <button>Download my full resume</button>
+        </div>
+      </div>
+
+      <div id="banner1" class="banner1-wrapper">
+        <div id="banner1-cv-title" class="banner1-cv-title">
```

Comment:
For the `banner1` section, it's already got an id selector and the nested elements can be easily identified by a class name, so the `banner1-cv-title` id is kinda of redundant here.